woocommerce/woocommerce-ios
# [Woo POS] Update floating control

## Description


Updated position of the floating control and added line overlay when card reader is not connected

## Testing information


- Enter POS
- Check that card reader is not connected and floating control shows it
- Tap on Connect your reader
- Check that the reader is connected and the floating control shows the connected status

## Screenshots


https://github.com/user-attachments/assets/8b189245-75ec-4fb1-85e2-8532a35432ba

---
